src/hits/hits.ts

```diff
@@ -35,30 +47,31 @@ import { NgAisInstantSearch } from '../instantsearch/instantsearch';
 export class NgAisHits extends BaseWidget {
   @ContentChild(TemplateRef) public template?: TemplateRef<any>;
 
-  // rendering options
-  @Input() transformItems?: Function;
+  @Input()
+  public transformItems?: <U extends HitsItem>(items: HitsItem[]) => U[];
 
-  // inner widget state returned from connector
-  public state: { hits: {}[]; results: {} } = { hits: [], results: {} };
+  public state: HitsState = {
+    hits: [],
+    results: {},
+  };
 
   constructor(
     @Inject(forwardRef(() => NgAisInstantSearch))
     public instantSearchParent: any
   ) {
     super('Hits');
-    this.createWidget(connectHitsWithInsights, { escapeHits: true });
+  }
+
+  ngOnInit() {
+    this.createWidget(connectHitsWithInsights, {
+      escapeHits: true,
```

Comment:
The connector option is named [`escapeHTML`](https://github.com/algolia/instantsearch.js/blob/49a5b64e42deec3453669d7dce788395cf898ba8/src/connectors/hits/connectHits.js#L57) since InstantSearch.js 3 and defaults to `true`. We can therefore remove this line.

Reply:
Good catch, I'll add this to this PR, it should not add too much load

Reply:
remove here: 50a6bb89e1a5b3393b14491b1c51bc73269891eb